Classes/Domain/Repository/SysFileRepository.php

```diff
@@ -92,7 +93,8 @@ public function findFromOracle(): array
                 $queryBuilder->expr()->eq('f.uid', $queryBuilder->quoteIdentifier('m.file'))
             )
             ->where(
-                $queryBuilder->expr()->isNotNull(self::FIELD_ASSET_ID)
+                $queryBuilder->expr()->isNotNull(self::FIELD_ASSET_ID),
+                $queryBuilder->expr()->eq('m.sys_language_uid', 0)
```

Comment:
Will always just fetch the default language?

Reply:
Yup. We don't receive localized data.